graphql/util.go

```diff
@@ -764,6 +752,25 @@ func ModifyVersionHandler(ctx context.Context, dataConnector data.Connector, pat
 	if err != nil {
 		return mapHTTPStatusToGqlError(ctx, httpStatus, err)
 	}
+
+	if modifications.Action != Restart {
```

Comment:
can we include  the reason you listed in the description for restart being different as a comment here? And can we check if the version requester is for a patch before running the patch query? (Then we can error if p == nil also)

Reply:
Also i'm assuming there's no such thing as nested child patches, in which case we could also check that ParentPatchID == "" so we don't call this unnecessarily for child patches.

Reply:
done. 

> we could also check that ParentPatchID == ""
I wrapped it in `if p.IsParent()`

Reply:
oh gotcha i misunderstood